common/src/main/java/org/opensearch/sql/common/setting/Settings.java

```diff
@@ -25,6 +25,7 @@ public enum Key {
 
     /** PPL Settings. */
     PPL_ENABLED("plugins.ppl.enabled"),
+    PPL_QUERY_TIMEOUT("plugins.ppl.query.timeout"),
```

Comment:
Does this impact both SQL and PPL queries or only PPL?

Reply:
I only implemented it with Calcite in mind, but I think it applies to both (up to whatever hits OSQueryManager)